src/constants/featureFlagsConstants.js

```diff
@@ -23,7 +23,10 @@ export const ENABLE_FEATURE_FLAG = 'ENABLE_FEATURE_FLAG';
 export const DISABLE_FEATURE_FLAG = 'DISABLE_FEATURE_FLAG';
 export const INITIAL_FEATURE_FLAGS = {
   SMART_WALLET_ENABLED: false,
+  // TODO: review 'bitcoin coming soon' message on accounts list
```

Comment:
is this sorted?

Reply:
Nope. This is for when BTC goes live. I've updated the text so it's a bit clearer, but not sure if it makes sens now. Let me know.